PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -797,31 +796,17 @@ private extension PanModalPresentationController {
      because we render the dragIndicator outside of view bounds
      */
     func addRoundedCorners(to view: UIView) {
-
-        let path = UIBezierPath()
-        path.move(to: CGPoint(x: 0, y: Constants.cornerRadius))
-
-        // 1. Draw left rounded corner
-        path.addArc(withCenter: CGPoint(x: path.currentPoint.x + Constants.cornerRadius, y: path.currentPoint.y),
-                    radius: Constants.cornerRadius, startAngle: .pi, endAngle: 3.0 * .pi/2.0, clockwise: true)
+        let radius = presentable?.cornerRadius ?? 0
+        let path = UIBezierPath(roundedRect: view.bounds,
+                                byRoundingCorners:[.topRight, .topLeft],
+                                cornerRadii: CGSize(width: radius, height: radius))
 
         // 2. Draw around the drag indicator view, if displayed
```

Comment:
nit: can we remove the numbered points now that we've cut down the complexity of this func 😄 i.e. `2.`, `6.` & `7.`